packages/ui/src/stellar/MintableSection.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import ExpandableToggleRadio from '../common/ExpandableToggleRadio.svelte';
+  import HelpTooltip from '../common/HelpTooltip.svelte';
+
+  export let mintable: boolean;
+  export let sequential: boolean;
+  export let consecutive: boolean;
+
+  let previousConsecutive = consecutive;
+  function handleSequentialChange(value: boolean) {
+    sequential = value;
+    if (!mintable) {
+      mintable = true;
+    }
+  }
+
+  $: if (consecutive !== previousConsecutive && consecutive) {
+    sequential = false;
+    mintable = false;
+  }
+
+  $: if (!mintable) {
+    sequential = false;
+  }
+
+  $: previousConsecutive = consecutive;
+</script>
+
+<ExpandableToggleRadio
+  label="Minting"
+  bind:value={mintable}
+  defaultValue={true}
+  helpContent="Configure minting mode for NFTs."
+  disabled={consecutive}
+>
+  <div class="checkbox-group">
+    <label class:checked={!sequential && mintable}>
+      <input
+        type="radio"
+        name="minting-mode"
+        value="non_sequential"
+        checked={!sequential && mintable && !consecutive}
+        on:change={() => handleSequentialChange(false)}
+        disabled={consecutive || !mintable}
```

Comment:
Minor preference, we don't necessarily need to disable the radio buttons if mintable is false.  This way, clicking them will cause mintable to become true.
```suggestion
        disabled={consecutive}
```

Reply:
If that is not a deal-breaker, I'd like to keep them as is :)

Reply:
It would differ a bit from other pages, but I think this is fine.